src/Examples/Examples.jl

```diff
@@ -19,6 +20,7 @@ using LinearAlgebra
 
 include("docs_examples.jl")
 include("other_examples.jl")
+include("ExamplesRepeatingStructures.jl")
```

Comment:
should follow the naming convention ie `repeating_structure_examples.jl`

Reply:
what was the reasoning behind this? It's clearly what we are doing but it seems strange that everywhere else we capitalize and do not use _ but in the examples files we use all lowercase and _. Is there some good reason for doing this? Or maybe we should make the example files follow the naming convention of the rest of the repo.

Reply:
The convention is for files containing modules to be capitalised and for ‘scripts’ to be lowercase with underscores